include/libtorrent/kademlia/node.hpp

```diff
@@ -134,7 +134,7 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	int data_size() const { return int(m_storage.num_torrents()); }
 #endif
 
-#ifndef TORRENT_DISABLE_LOGGING
+#if TORRENT_USE_IOSTREAM
 	// TODO: 3 make this print to the DHT logger instead
```

Comment:
I'm thinking of this TODO comment. this change seems to be moving in the opposite direction. I'm not married to this idea if you have a good reason to move in this direction